### DISCUSSION TITLE
Change Backup filename

### DISCUSSION THREAD

**Discussion by KareemWaheed** [tiredofit/docker-db-backup#450] [Ideas]
Hi, I was wondering if there is a possibility of change the backup file name/naming format to a custom name 

I'm uploading backups to S3 and i could enable versioning so that latest backup always has one name that i could restore via a script if needed without changing the name

Thank you

**Comment by tiredofit** [comment]
At this time there is no way to format the file - I do some pretty hairy stuff with the filenames based on what type of compression and databases are taken. I can see how it can happen but would have to map out what it would take to get working properly without clobbering any of the backend work.

**Comment by cdpb** [comment]
It wouldn't be hard to workaround that with a post backup script since it reports the filename. Just `ln` it to `latest` and you should be good to go. Just my educated guess, not tested though.

**Comment by tiredofit** [comment]
This is a great way of utilizing post script! Lovely idea.
